Package.swift

```diff
@@ -17,7 +17,7 @@ let package = Package(
     )
   ],
   dependencies: [
-    .package(url: "https://github.com/apple/swift-collections", from: "0.0.1"),
+    .package(url: "https://github.com/apple/swift-collections", from: "0.0.7"),
```

Comment:
While it probably doesn't _hurt_ to bump this number, we've been trying to avoid forced upgrades of external libraries, especially if APIs are backwards compatible (someone may depend on another library that depends on `swift-collections` with stricter versioning requirements, and we don't want to trample on that if we can avoid it).

I _think_ `elements` has been public for awhile, so it should be safe to leave this at `"0.0.1"` for now.

Reply:
Good call! Thank you 🙏 Didn't see this was there all along